pkg/statestore/leveldb/migration.go

```diff
@@ -34,10 +34,11 @@ const (
 	dbSchemaDrain         = "drain"
 	dbSchemaCleanInterval = "clean-interval"
 	dbSchemaNoStamp       = "no-stamp"
+	dbSchemaIdk           = "idk"
```

Comment:
more descriptive name?

Reply:
i was actually hoping for codenames instead of descriptive names. now it is a mixture of both which is meh